rsocket-transport-netty/src/main/java/io/rsocket/transport/netty/TcpDuplexConnection.java

```diff
@@ -60,11 +67,19 @@ public boolean isDisposed() {
 
   @Override
   public Mono<Void> send(Publisher<Frame> frames) {
-    return Flux.from(frames).concatMap(this::sendOne).then();
+    return Flux.from(frames)
+        .map(Frame::content)
+        .flatMapSequential(this, 256, Integer.MAX_VALUE)
+        .then();
   }
 
   @Override
   public Mono<Void> sendOne(Frame frame) {
-    return connection.outbound().sendObject(frame.content()).then();
+    return outbound.sendObject(frame.content()).then();
+  }
+
+  @Override
+  public Publisher<Void> apply(ByteBuf frame) {
```

Comment:
What is the utility of this?  Also noticeable that because of the internal implementation, this one is

apply(ByteBuf), sendOne(Frame)

While WS is

apply(Frame), sendOne(Frame)

Reply:
Think this comes from https://github.com/rsocket/rsocket-java/pull/549/files